test/e2e-go/cli/algod/expect/algod_expect_test.go

```diff
@@ -19,12 +19,11 @@ import (
 	"testing"
 
 	"github.com/algorand/go-algorand/test/framework/fixtures"
-	"github.com/algorand/go-algorand/test/partitiontest"
 )
 
 // TestAlgodWithExpect Process all expect script files with suffix Test.exp within the test/e2e-go/cli/algod/expect directory
 func TestAlgodWithExpect(t *testing.T) {
-	partitiontest.PartitionTest(t)
+	// partitiontest.PartitionTest(t) Since each expect test is assigned a partition in `et.Run`, avoid partitioning here.  Creates double partitioning.
```

Comment:
I recommend this wording to match the other exampled (Algoh and Goal).
```suggestion
	// partitiontest.PartitionTest(t)
	// Causes double partition, so commented out on purpose
```